randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -886,7 +886,12 @@ Extra - asset_id: collision_camera_013
   * Layers: default
   * Morph Ball Tunnel to Space Jump Room/Tunnel to Pitfall Puzzle Room (Upper)
   > Pickup (Missile Tank)
-      Morph Ball and After Ferenia - Pitfall Puzzle Blob and Use Spin Boost
+      All of the following:
+          Morph Ball and After Ferenia - Pitfall Puzzle Blob
+          Any of the following:
+              Use Spin Boost
+              # With Flash Shift: https://youtu.be/XBcdDthTfdg
+              Flash Shift and Movement (Intermediate)
```

Comment:
This is maybe fine, but for reference, look at the item at the top of Ghavoran Spin Boost Tower. Base on that, these should have "Climb Sloped Tunnels".